app/src/gui/components/workspace/notebooks.tsx

```diff
@@ -194,17 +198,35 @@ export default function NoteBooks() {
           </Button>{' '}
           tab and click the activate button.
         </Typography>
-        {showCreateNewNotebookButton && (
+        <div
+          style={{display: 'flex', justifyContent: 'space-between', gap: '8px'}}
+        >
+          {showCreateNewNotebookButton ? (
+            <Button
+              variant="contained"
+              onClick={() => history(ROUTES.CREATE_NEW_SURVEY)}
+              sx={{mb: 3, mt: 3, backgroundColor: theme.palette.primary.main}}
+              startIcon={<AddCircleSharpIcon />}
+            >
+              Create New {NOTEBOOK_NAME}
+            </Button>
+          ) : (
+            <div />
```

Comment:
why are you rendering an empty div?

Reply:
empty div pushes the other button to the right in the flex box due to `justifyContent: 'space-between'`. Could change to `justifySelf: 'end'` on the child but I generally default to putting in the parent.